kong/plugins/azure-functions/handler.lua

```diff
@@ -61,17 +45,33 @@ function azure:access(config)
   local s = find(upstream_uri, "?", 1, true)
```

Comment:
A comment describing what all the string manipulations below do would be much appreciated.